.config/webpack/base.js

```diff
@@ -19,29 +19,19 @@ module.exports = (options = {}) => {
     return {
         entry: {
             bundle: './src/dash-table/index.ts',
-            demo: ['./demo/index.js', './demo/index.html']
+            demo: ['./demo/index.html', './demo/index.js']
         },
         mode: mode,
         output: {
             path: path.resolve(__dirname, `./../../${dashLibraryName}`),
             filename: '[name].js',
             library: dashLibraryName,
-            libraryTarget: 'umd'
+            libraryTarget: 'window'
```

Comment:
Loaded unto `window.dash_table` in the browser

Reply:
Will changing how this is exposed have any side-effects for non-Dash users? Could you imagine anyone using RequireJS or anything else that expects a `umd` in their JavaScript project? I suppose, obviously, we wouldn't expect Node.js users (without a `window` global namespace) to be using this project. I'm also not sure if there's any webpack magic that makes this a non-issue.

Reply:
This should not have an impact.